WORKSPACE

```diff
@@ -39,4 +39,6 @@ gcc_register_toolchain(
     target_arch = "armv7",
     binary_prefix = "arm",
     platform_directory = "arm-buildroot-linux-gnueabihf",
+    hardcode_sysroot_ld_linux = False,
```

Comment:
how come we don't use the new feature ourselves?

Reply:
We do. This is for armv7. We don't want to do this when cross-compiling.